src/EditorFeatures/Core/Implementation/CallHierarchy/Finders/OverridingMemberFinder.cs

```diff
@@ -24,7 +24,7 @@ public override string DisplayName
         {
             get
             {
-                return EditorFeaturesResources.Overrides;
+                return EditorFeaturesResources.Overrides_;
```

Comment:
Overrides is a VB keyword.  My tool avoids keywords so there won't be a problem on the consumption side.  It does this by adding _ at the end.